p0_user_stories.md

```diff
@@ -31,22 +31,22 @@ other pods should be blocked from accessing it via any IP.
 
 - I want all namespaces matching X to be completely 100% locked-down by default; no pods can talk to any other pods until the developers specify policies allowing it - I may also want the converse situation for namespace Y.
 
-- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all my clusters have different Service CIDRs.
+- I want to target all apps with network policies, but most of my apps need apiserver access, and i dont know the k8s service IP because all of my clusters have different Service CIDRs.
```

Comment:
Does this assume that the apiserver is an actual pod within the cluster? How would we do this if it is not within the cluster?